lib/navigation/main.cpp

```diff
@@ -0,0 +1,26 @@
+#include "main.hpp"
+
+//TODO: move/ rename/ make this less bad
+
+namespace hyped::navigation{
+
+  Main::Main(){
+    //TODO: this
+  }
+
+  void Main::navigate(){
```

Comment:
I don't think we need a main at all?

This is supposed to be a library, i.e. it needs to have all the nontrivial functionality that is not related to ROS.

In `src/` there will then be a node file which instantiates a particular navigator and does navigation.

Reply:
+1

Reply:
Yeah realised this at the end of the meeting. main.[ch]pp will be leaving us shortly